tardis/model/tests/test_csvy_model.py

```diff
@@ -75,6 +76,26 @@ def test_compare_models(model_config_fnames):
     )
 
 
+def test_dimensionality_after_update_v_inner_boundary(
+    model_config_fnames,
```

Comment:
See if there are fixtures that loads the model.

Reply:
If there's isn't already a fixture for this, we should write one. In general I think unless there's a very good reason for it, no test should reload these generic objects or the full list of tests will reload things many times without much point.

Reply:
created new fixture in this test.py